x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/exceptions/shared_exception_lists_management/shared_exception_list_page/manage_lists.cy.ts

```diff
@@ -56,7 +60,9 @@ describe(
   { tags: ['@ess', '@serverless'] },
   () => {
     describe('Create/Export/Delete List', () => {
-      before(() => {
+      beforeEach(() => {
+        deleteAlertsAndRules();
+        deleteExceptionLists();
```

Comment:
Seeing the same pattern as above here: after the last test, any rules and exception lists created will still be present for the next test. There should probably also be an `afterEach` added with these commands.

Trying to set up a pristine state in the `beforeEach` is a good, defensive practice, but cleaning up your own test's data is much more important. I suspect that's been the cause of some flakiness in our suite.